### DISCUSSION TITLE
Can I use bouncycastle 1.72 instead of 1.70 for web3j v4.9.7

### DISCUSSION THREAD

**Discussion by soharang** [web3j/web3j#1894] [Q&A] [answered]
Hello, 

I am using several 3rd party library including web3j.
Now a days, I got a compile error like following.
![image](https://user-images.githubusercontent.com/32191478/231397209-a02e14e1-9834-4b55-af9d-d2fd725f6eec.png)

Bouncycastle v1.70 is used by web3j and bouncycastle v1.72 is used by other library.
So I would like to modify one of library version to another - 1.70 or 1.72.

Do you think web3j v4.7.2 is OK with bouncycastle v1.72?

I am also asking same question to them.

Thank you.

**Comment by NickSneo** [comment, marked as answer]
Yes it should be fine. Also in the latest Web3j version, most of the libraries are updated. So you can use Web3j 4.10.3 which uses bouncycastle v1.73
